src/templates/base/base.html

```diff
@@ -777,6 +777,7 @@
 {# Ebook needs a unique fig id. 2019 is already out there so reluctant to change that, so "if" statemnt it is! #}
 {% if ebook %}
 {% set anchor="fig-%s-%s" % (metadata.chapter_number, id) %}
+{% if link.startswith("/") %}{% set link="https://almanac.httparchive.org%s" % link %}{% endif %}
```

Comment:
Does this account for relative URLs that start with `./` for example?

Reply:
We don't currently have any of those. Can see how we would have to be honest as then they would be relative to the chapters not the static image directory.

Guess in theory could link to another chapter from an image but don't see why you'd do that.

I see leave for now as relative paths would be more complex to support. It's not the end of the world if the link is wrong - which is why no one's notice it's been broken for a while! The image still displays, just link broken if you click on it while reading PDF.